doc/elements.md

```diff
@@ -481,6 +481,7 @@ Attribute | Type | Default | Description
 --- | --- | --- | ---
 `language` | string | — | The programming language syntax highlighting to use. See below for options.
 `no-highlight` | boolean | false | Disable highlighting.
+`input-file-name` | string | None | Name of the file with content to be displayed inside the code snippet (instead - or in addition to - writing the text between the open/closing tags).
```

Comment:
I'd consider using `source-file-name` instead - `input` is a very overloaded term in these parts 🙂